app/controllers/ResultsPageController.scala

```diff
@@ -16,18 +16,19 @@
 
 package controllers
 
-import connectors.MarginalReliefCalculatorConnector
+import connectors.{ CalculatorConfigConnector, MarginalReliefCalculatorConnector }
 import controllers.actions._
-import forms.{ AccountingPeriodForm, AssociatedCompaniesForm, DistributionsIncludedForm }
+import models.ResultsPageData
+import models.requests.DataRequest
 import pages.{ AccountingPeriodPage, AssociatedCompaniesPage, DistributionsIncludedPage, TaxableProfitPage }
 import play.api.i18n.{ I18nSupport, MessagesApi }
 import play.api.mvc.{ Action, AnyContent, MessagesControllerComponents }
 import uk.gov.hmrc.http.BadRequestException
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
-import views.html.ResultsPageView
+import views.html.{ FullResultsPageView, ResultsPageView }
 
 import javax.inject.Inject
-import scala.concurrent.ExecutionContext
+import scala.concurrent.{ ExecutionContext, Future }
 
 class ResultsPageController @Inject() (
```

Comment:
As the full page is a new view, probably worth moving the code to a new controller - FullResultsPageController?